portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,101 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Hide the comments form by default and fetch the login status from the servlet
+ * If the user is logged in, unhide the form
+ * If the user is not logged in, display a login link
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function checkLoginStatus(){
+    fetch('/home') // Sends a request to /home
+    .then(response => response.json()) // Parse the response as JSON
+    .then((object) => { // Reference the status in response
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+        const status = object.status;
+        const url = object.url;
+
+        if (status == 'true') {
+            document.getElementById('comment').style.display = 'block';
+            var text = 'logouthere';
+        }
+        else {
+            var text = 'loginhere';
+        }
+
+        // Create the button
+        var button = document.createElement('button');
+        button.setAttribute('id','logbtn');
+        button.innerHTML = String(text);
+
+        // Append in login div
+        var login = document.getElementById('login');
+        login.appendChild(button);
+
+        // Add event handler
+        button.addEventListener('click', 
+        function() {
+            window.location.href = url;
+        });
```

Comment:
I think this should be indented more to the right.

Reply:
The indentation is added.